operations/finance.md

```diff
@@ -797,14 +803,21 @@ Closes a bill so no further modification to it is possible.
 | `Type` | string [Bill type](#bill-type) | required | Specifies the mode bill should be closed in. |
 | `BillCounterId` | string | optional | Unique identifier of the [Counter](enterprise.md#counter) to be used for closing. Default one is used when no value is provided. |
 | `FiscalMachineId` | string | optional | Unique identifier of the [Fiscal Machine](integrations.md#device) to be used for closing. Default one is used when no value is provided. |
-| `Options` | array of string [Bill options](#bill-options) | optional  | Options of the bill, only `DisplayCustomer` and `DisplayTaxation` can be used. If not provided both `DisplayCustomer` and `DisplayTaxation` are set. |
-| `TaxedDate` | string | optional | Date of consumption for tax purposes. Can be used only with [Bill type](#bill-type) `Invoice`. |
-| `DueDate` | string | optional | Deadline when [Bill](#bill) is due to be paid. Can be used only with [Bill type](#bill-type) `Invoice`. |
-| `VariableSymbol` | string | optional | Optional unique identifier of requested payment. Can be used only with [Bill type](#bill-type) `Invoice`. |
-| `TaxIdentifier` | string | optional | Tax identifier of account to be printed on a bill. |
-| `Notes` | string | optional | Notes to be attached to bill. |
+| `Options` | [Bill options parameters](#bill-options-parameters) | optional  | Options of the bill. If not provided both `DisplayCustomer` and `DisplayTaxation` are set by default. |
+| `TaxedDate` | [String update value](reservations.md#string-update-value) | optional | Date of consumption for tax purposes. Can be used only with [Bill type](#bill-type) `Invoice`. |
+| `DueDate` | [String update value](reservations.md#string-update-value) | optional | Deadline when [Bill](#bill) is due to be paid. Can be used only with [Bill type](#bill-type) `Invoice`. |
+| `VariableSymbol` | [String update value](reservations.md#string-update-value) | optional | Optional unique identifier of requested payment. Can be used only with [Bill type](#bill-type) `Invoice`. |
+| `TaxIdentifier` | [String update value](reservations.md#string-update-value) | optional | Tax identifier of account to be printed on a bill. |
+| `Notes` | [String update value](reservations.md#string-update-value) | optional | Notes to be attached to bill. |
 | `Address` | [Address parameters](customers.md#address-parameters) | optional | Address of the account to be displayed on bill. Overrides the default one taken from account profile. |
 
+#### Bill options parameters
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `DisplayCustomer` | [Bool update value](reservations.md#bool-update-value) | required | Display customer information on printed bill. |
```

Comment:
Let's completely omit the word "printed" because it also applies to PDF or just viewing the bill in navigator UI.